src/cli/commands/app.rs

```diff
@@ -0,0 +1,17 @@
+use clap::Subcommand;
+
+#[derive(Subcommand)]
+pub(crate) enum Commands {
```

Comment:
Not blocking this PR — do we also want a way to list the apps?

Reply:
Yes, though there’s no support for that in the API at this time. I also have to expose /edit APIs much like what you can do in the web UI